.github/workflows/cd.yml

```
@@ -1,158 +1,158 @@
-
-              echo "==> recent deploy log"
-              tail -n 300 /var/log/transcendence-deploy.log || true
-            '
-
-      - name: Fail workflow if external smoke failed
-        if: steps.external_smoke.outcome == 'failure'
-        run: |
-          echo "External smoke check failed"
-          exit 1
\ No newline at end of file
+#name: CD — Deploy to Production
```

actionlint (1.7.12)

[error] workflow is empty

(syntax-check)